addon/components/task-button.hbs

```diff
@@ -0,0 +1,21 @@
+<WorkItemButton
+  ...attributes
+  @mutation={{@mutation}}
+  @label={{@label}}
+  @workItemId={{this.workItemId}}
+  @type={{@type}}
+  @disabled={{@disabled}}
+  @active={{@active}}
+  @loading={{@loading}}
+  @color={{@color}}
+  @size={{@size}}
+  @title={{@title}}
```

Comment:
Dont forget to pass on `onError` and `onSuccess`

Reply:
Changed